docs/src/architectures/pet.rst

```diff
@@ -8,9 +8,94 @@ PET
   This is an **experimental model**.  You should not use it for anything important.
 
 
-Architecture Hyperparameters
+PET Hyperparameters
 ----------------------------
-:param name: ``experimental.pet``
+
+There are two significant groups of hyperparameters controlling PET fits. The \
```

Comment:
you should not need any `\` here, rst does not care if your paragraph spans multiple lines